src/Nest/XPack/AsyncSearch/AsyncSearchResponseBase.cs

```diff
@@ -1,11 +1,15 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
 using System;
 using System.Runtime.Serialization;
 using Elasticsearch.Net.Utf8Json;
 
 namespace Nest
 {
 	[InterfaceDataContract]
-	public interface IAsyncSearchResponse<TDocument> : IResponse where TDocument : class
```

Comment:
This is a breaking change we can't do in `7.11` since this was already released `7.7`